components/CreateBloks.tsx

```diff
@@ -1,6 +1,12 @@
 import { StoryblokComponent } from '@storyblok/react/rsc';
+import { SbBlokData } from '@storyblok/react/rsc';
```

Comment:
```suggestion
import { StoryblokComponent, type SbBlokData } from '@storyblok/react/rsc';
```

Reply:
Thanks! I didn't know this trick. I'll update it everywhere 😄